src/utils/cookies.js

```diff
@@ -125,11 +125,11 @@ export const saveCookiePreferences = () => {
     state._allCategoryNames.forEach(categoryName => {
 
         state._lastChangedServices[categoryName] = arrayDiff(
-            state._enabledServices[categoryName],
+            state._enabledServices[categoryName] || [],
```

Comment:
This should not be needed since `state._enabledServices[<category>]` is always a valid array

Reply:
`state._enabledServices` is a valid array for the previously enabled category names, but if you change a category name or add a new one in config, it won't be in the array.